docs/fundamentals/code-analysis/style-rules/ide0110.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "IDE0110: Discard can be removed"
+description: "Learn about code analysis rule IDE0110: Discard can be removed"
+ms.date: 11/02/2020
+ms.topic: reference
+f1_keywords:
+- IDE0110
+helpviewer_keywords:
+- IDE0110
+author: CyrusNajmabadi
```

Comment:
It is fine to put your alias if you plan to maintain/fix this doc page. Otherwise, I would recommend changing the author and ms.author to `gewarren`, so the docs team owns it going forward.

Reply:
Related tip: If you leave them blank, the default owner (in this case @gewarren) will be used.

Reply:
sgtm.